tools-local/tasks/mobile.tasks/WasmBuildTasks/CompileTimeZoneData.cs

```diff
@@ -81,17 +84,21 @@ public override bool Execute()
             Log.LogError($"Could not find required file {ZoneTabFile}");
             return false;
         }
-
         CompileTimeZoneDataSource();
 
-        string[] filtered = new string[] { "America/Los_Angeles", "Australia/Sydney", "Europe/London", "Pacific/Tongatapu",
+        if (FilterSystemTimeZones!) {
+            string[] filtered = new string[] { "America/Los_Angeles", "Australia/Sydney", "Europe/London", "Pacific/Tongatapu",
                                 "America/Sao_Paulo", "Australia/Perth", "Africa/Nairobi", "Europe/Berlin",
                                 "Europe/Moscow", "Africa/Tripoli", "America/Argentina/Catamarca", "Europe/Lisbon",
                                 "America/St_Johns"};
+            FilterZoneTab(filtered, ZoneTabFile);
+        }
+        else
+        {
+            File.Copy(ZoneTabFile, Path.Combine(OutputDirectory!, "zone.tab"), true);
```

Comment:
Is this *new* intended behavior, to have `zone.tab` in the output directory, as the output of the task?

Reply:
Yes it is intended to have that file in the output directory so that it is bundled into `dotnet.timezones.blat`